pkg/vmprovider/providers/vsphere/virtualmachine/backup.go

```diff
@@ -37,6 +48,11 @@ func BackupVirtualMachine(
 		vmCtx.Logger.Info("No VM bootstrap data is provided for backup")
 	}
 
+	vmDiskData, err := getEncodedVMDiskData(vmCtx.Context, vcVM)
```

Comment:
Can just use `vmCtx` here instead. Ignore comment if you intentionally did this for readability or another reason.